lib/wsdl.js

```diff
@@ -2132,17 +2139,21 @@ function open_wsdl(uri, options, callback) {
   var wsdl;
   if (!/^https?:/.test(uri)) {
     debug('Reading file: %s', uri);
-    fs.readFile(uri, 'utf8', function(err, definition) {
-      if (err) {
-        callback(err);
-      }
-      else {
-        wsdl = new WSDL(definition, uri, options);
-        WSDL_CACHE[ uri ] = wsdl;
-        wsdl.WSDL_CACHE = WSDL_CACHE;
-        wsdl.onReady(callback);
-      }
-    });
+    if (typeof uri !== "undefined") {
```

Comment:
why just check for `=== 'undefined'`? this would fail if `uri` is `null`... wouldn't it be better to check for `typeof uri === 'string'` in order to prevent the `fs.readFile()` to fail?